pegasus/sites.v3/code.org/public/translate/frozen.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Translate Frozen"
+nav: translate_nav
+theme: responsive
+---
+
+# Translating Code with Anna and Elsa
+Translations for the [Code with Anna and Elsa](https://studio.code.org/s/frozen) tutorial are done on Crowdin and translations for the video captions are done on Amara. They are both free to join and any volunteer can immediately start translating on either platform.
```

Comment:
Let's make this URL relative instead of absolute. We have a few environments like staging-studio.code.org where we see a preview of the latest Code.org's changes before they go live on production at studio.code.org. So using a relative link helps keep people on the same environment when they're clicking around. We have a special method you can use called `CDO.studio_url` that will prepend the URL with the environment's correct domain. For example, staging-studio.code.org when you're on staging or test-studio.code.org when you're on test. It will be embedded Ruby so try including it like this:

`[Code with Anna and Elsa](<%= CDO.studio_url('/s/frozen') %>)`